### PR TITLE
Cleanup in 03_Navigation.md: Whitespace/indentation, consistent parameter keys, …

### PR DESCRIPTION
**Pull request by NiklasBr**
## Changes in this pull request  
Just some smaller changes.

### CODE REVIEW

File: doc/Development_Documentation/03_Documents/03_Navigation.md [hunk 1]
```diff
@@ -287,85 +285,78 @@ use Twig\TwigFunction;
...
+     */
+    public function getFunctions()
+    {
+        return [
+            new TwigFunction('app_navigation_news_links', [$this, 'getDataLinks'])
+        ];
+    }
+
+    /**
+     * @param Document $document
```
Comment (dvesh3) [inline doc/Development_Documentation/03_Documents/03_Navigation.md:312]: we don't add phpdocs, if there are typehints present. please remove it.

Comment (NiklasBr) [reply doc/Development_Documentation/03_Documents/03_Navigation.md:312]: This one was missing from the ones added already.

**Review by fashxp — APPROVED** [review]

**Comment by fashxp** [comment]
thx very much

**Comment by blankse** [comment]
The milestone should be changed to 10.5.22

**Pull request merged**